### PR TITLE
Week 2 parallax

### PR DESCRIPTION
**Pull request by mmurray22**
This PR is centered around the parallax functionality. The relevant pieces of code will all be within the script.js file and include both the variables declared at the top of the file and the functionality within the $(document).ready call.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -5,63 +5,65 @@
...
+                <li>Stanford Class of 2022</li> 
+                <li>Major: Computer Science</li>
+                <li>STEP Intern @ Google</li>
+                </ul>
+            </div>
+        </div>
+        </div>
+        <div id="intro-button-two" style="display: inline-block;">
+        <div class="mdc-touch-target-wrapper" style="display: block;">
+            <button class="mdc-button mdc-button--outlined" onclick="blog()" style="color:white;">
```
Comment (ccondit) [inline portfolio/src/main/webapp/index.html:50]: nit - revert this change related to the blog.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -5,63 +5,65 @@
...
-        </div> 
+                <span class="mdc-button__ripple">Resume</span>
+            </button>
+            </form>
+        </div>
         <div class="mdc-touch-target-wrapper" style="display:block;">
-      	  <button class="mdc-button mdc-button--outlined" onclick="contact()" style="color:white;">  
-	     <span class="mdc-button__ripple">Contact</span> 
-	  </button>
+            <button id="picture-button" class="mdc-button mdc-button--outlined" onclick="pictures()" style="color:white;">
```
Comment (ccondit) [inline portfolio/src/main/webapp/index.html:62]: nit - revert this change related to pictures and send in a separate PR.

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -5,63 +5,65 @@
     <!-- Imported Libraries: Google's Material Design Library and Google's Fonts Library -->
     <link href="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.css" rel="stylesheet">
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```
Comment (ccondit) [inline portfolio/src/main/webapp/index.html:8]: Is jquery used in this PR?

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -12,9 +12,39 @@
...
+
+  $("#profile").on('mousemove click', function(e) {
+        var lMouseX = Math.max(-100, Math.min(100, $('#profile').width()*2 - e.clientX));
+        var lMouseY = Math.max(-100, Math.min(100, $('#profile').height() / 2 - e.clientY));
+        lFollowX = (12 * lMouseX) / 100; // 100 : 12 = lMouxeX : lFollow
+        lFollowY = (10 * lMouseY) / 100;
+        /*Animate!*/
+        animate();
+  });
+});
```
Comment (ccondit) [inline portfolio/src/main/webapp/script.js:41]: nit - I *think* all the changes below this file are unrelated to the parallax work - can you revert them in this PR?

**Review by ccondit — CHANGES_REQUESTED** [review]
nit - also revert changes unrelated to parallax in the css.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/index.html:50]: I took out the blog function! (I believe the button was there before) Thanks!

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/index.html:62]: This change falls under the formatting PR-I will remove this from this PR thanks!

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/index.html:8]: Yes

File: portfolio/src/main/webapp/script.js (hunk 4 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/script.js:41]: Yes they are-I removed the blog() and picture() functions, but all the previous functions were there in the previous PR-should I remove those too?
